v2/charts/azure-service-operator/templates/generated/cert-manager.io_v1_certificate_azureserviceoperator-serving-cert.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: azureserviceoperator-serving-cert
+  namespace: {{.Release.Namespace}}
+spec:
+  dnsNames:
+  - azureserviceoperator-webhook-service.{{.Release.Namespace}}.svc
```

Comment:
Yep, helm always overrides the namespace by .Release.Namespace or default(if release namespace not specified). So, rather than putting something else which is anyway going to get overridden, we have to use `{{.Release.namespace}}`.